examples/pytorch/ogb/ogbn-mag/hetero_rgcn.py

```diff
@@ -14,44 +16,20 @@ def extract_embed(node_embed, input_nodes):
     emb = {}
     for ntype, nid in input_nodes.items():
         nid = input_nodes[ntype]
-        if ntype in node_embed:
-            emb[ntype] = node_embed[ntype][nid]
+        if ntype in node_embed.embeds:
+            emb[ntype] = node_embed.embeds.pop(ntype)(nid)
```

Comment:
Do we need to "pop" it? Perhaps it's enough to do `node_embed.embeds[ntype]`?

Reply:
The proper code may be "node_embed.embeds[ntype]（nid）“。The test results are as follows:
![image](https://user-images.githubusercontent.com/52376036/183081116-50b47d83-cf72-4d2e-abba-41305ae4d56f.png)

Reply:
Yes. That's what I meant.